### PR TITLE
Initial file for Travis CI

### PR DESCRIPTION
**Pull request by kchitalia**
I will add the tests in a separate pull request.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/kchitalia/SalesforceMobileSDK-iOS.svg?branch=unstable)](https://travis-ci.org/kchitalia/SalesforceMobileSDK-iOS)
```
Comment (bhariharan) [inline readme.md]: Shouldn't this be ```https://travis-ci.org/forcedotcom/SalesforceMobileSDK-iOS```? We should use the main repo. Also, could we move this to the bottom of this file?

**Comment by wmathurin** [comment]
Looks fine

**Comment by kchitalia** [comment]
@bhariharan 
1. I will change the link for the image to use the forcedotcom repo. 

2. Typically github repos have this at the beginning of the readme file so its easy to see the  build status as soon as you go to the repo page.

**Comment by khawkins** [comment]
:+1: to keeping it at the top, as it's displaying build status.

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**